_posts/2015-05-22-meetup-start-hacking.markdown

```diff
@@ -8,4 +8,10 @@ author: Liveplant Team
 
 [Meetup link for posterity](http://www.meetup.com/Dallas-Web-Mobile-Development-Meetup/events/222583236/)
 
-Everyone start adding some notes!
+Many team members were missing this meetup, so it was rather relaxed. Had the pleasure of meeting a new developer to the group and introducing him to Liveplant.
```

Comment:
Had the pleasure of adding a new developer to the group and introducing him to Liveplant.